backend/src/LeanCode.AppRating/DataAccess/ModelBuilderExtensions.cs

```diff
@@ -11,7 +12,11 @@ public static void ConfigureAppRatingEntity<TUserId>(this ModelBuilder builder,
     {
         builder.Entity<AppRating<TUserId>>(c =>
         {
-            c.HasKey(e => new { e.UserId, e.DateCreated });
+            c.HasKey(e => new { e.Id });
```

Comment:
```suggestion
            c.HasKey(e => e.Id);
```

Or remove this altogether because it matches EF convention.